docs/dev/table/connect.md

```diff
@@ -1284,7 +1284,11 @@ CREATE TABLE MyUserTable (
   'connector.username' = 'name',
   'connector.password' = 'password',
   
-  -- **followings are scan options, optional, used when reading from table**
+  -- **followings are scan options, optional, used when reading from a table**
+
+  -- optional: SQL query / prepared statement.
+  -- If set, it will takes over the 'connector.table' setting
+  'connector.read.query' = 'SELECT * FROM sometable',
```

Comment:
```suggestion
  -- optional: SQL query / prepared statement.
  -- If set, this will take precedence over the 'connector.table' setting
  'connector.read.query' = 'SELECT * FROM sometable',
```